xml/hwbp_registers.xml

```diff
@@ -175,6 +175,17 @@
 
             1: While this trigger does not match, it prevents the trigger with
             the next index from matching.
+
+            Because \Fchain affects the next trigger, hardware must zero it in
+            M mode writes to \Rmcontrol if the next trigger has \Fdmode of 1.
+            In addition hardware should ignore writes to \Rmcontrol with
+            \Fdmode of 1 if the previous trigger has both \Fdmode of 0 and
```

Comment:
I would phrase this as "writes to \Rmcontrol that would result in \Fdmode being 1".  As written, it's unclear whether it's talking about the old or new value of \Fdmode.

Reply:
How about "writes to \Rmcontrol that set \Fdmode to 1"?

Reply:
yeah